boards/arm/mimxrt1050_evk/doc/mimxrt1050_evk.rst

```diff
@@ -182,12 +186,25 @@ your Zephyr application to flash. It will leave you at a GDB prompt.
    :board: mimxrt1050_evk
    :goals: debug
 
+Board Revisions
+***************
+
+MIMXRT1050-EVKB is revison A1, while MIMXRT1050-EVK board is rev. A0. There are
+the following major hardware differences:
+
+- SoC changed from MIMXRT1052DVL6**A** to MIMXRT1052DVL6**B**
+- Hardware bug fixes for: power, interfaces, memory
+- Arduino headers included
+
+For more details, please see the following `application note`_.
```

Comment:
Change to:

    For more details, please see this `NXP i.MXRT1050 A0 to A1 Migration Guide`_,